skycultures/greek_farnese/description.de.utf8

```diff
@@ -0,0 +1,339 @@
+<h1>Farnese </h1>
+<p>Diese antike griechische
+Himmelskultur basiert auf dem Marmorglobus der Atlas Statue in der Villa Farnese (Neapel). Die Statue datiert auf das 2. Jahrhundert n. Chr., also in römische Zeit. Sie ist aber eine Kopie einer älteren griechischen Vorlage, die selbst nicht erhalten ist.</p>
+<p >
+ Der Marmorglobus bildet nur Figuren ab und keine Sterne. Der Globus hat einen Durchmesser von etwa 65 cm. Die Figuren wurden nicht nach genauen Punktkoordinaten ausgerichtet, sondern in einem Gitternetz: Ekliptik, Himmelsäquator, die Wendekreise und die Polarkreise geben eine Orientierung hinsichtlich der Lage in "Breitenkreisen" auf der Himmelskugel, während die Kolurkreise eine Vierteilung in "Längenkreisen" bieten. Antike Globenmachen nutzten dieses sehr grobmaschige Netz, um die Figurzeichnungen zu justieren. 
+ </p>
+ <p>
+ Die Abbildungen des Globus zeigen daher klar, dass der Steinmetz, der den Globus erstellte, das populäre Lehrgedicht von Aratos von Soloi im Kopf hatte. Es gibt keinen Grund für die - oft populärwissenschaftlich aufgebauschte - Behauptung, dass dieser Marmorglobus nach dem Sternkatalog von Hipparch gemeißelt worden wäre. Diese These wurden in den vergangenen ~200 Jahren wiederholt aufgeworfen, jedoch regelmäßig wieder verworfen: Sowohl mathematische Argumente, als auch künstlerische Darstellungen sprechen dagegen.  
```

Comment:
optional: add years for Aratos?

Reply:
exact years are always a unknown - but I see your point; added (~300BCE)